### DISCUSSION TITLE
Combine multiple entities into the same device

### DISCUSSION THREAD

**Discussion by Nawor3565** [home-assistant/frontend#9092] [Configuration]
After setting up both my ESPHome relay and AsusWRT, I noticed that the entities corresponding to the ESPhome relay from both integrations were combined into a single device:

![image](https://user-images.githubusercontent.com/19783632/117041016-7faeaf00-acd8-11eb-8fa5-109c98762392.png)

this is a great feature as it helps to keep clutter down by avoiding duplicate devices, of which I have quite a few (usually, one entity from AsusWRT, and at least one from the device's own integration). However, it seems like ESPHome devices are the only ones that do this. Would it be possible to add a feature where you can manually combine devices in this way? Thanks!

**Comment by frenck** [comment]
> However, it seems like ESPHome devices are the only ones that do this.

All devices that have the same identifier do this. This is not specific to ESPHome.

> Would it be possible to add a feature where you can manually combine devices in this way? Thanks!

That is not a frontend request, but a backend feature request. Please raise feature requests for the general core here:

<https://community.home-assistant.io/c/feature-requests/13>

Thanks! 👍
